app/Http/Controllers/SpellListController.php

```diff
@@ -17,25 +20,40 @@ public function index()
         return response()->json(["data" => SpellList::all()], 200);
     }
 
-    /**
-     * Show the form for creating a new resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function create()
-    {
-        //
-    }
-
     /**
      * Store a newly created resource in storage.
      *
      * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(SpellListStoreRequest $request)
     {
-        //
+        $spellList = new SpellList;
+
+        $validated = $request->validated();
+
+        if (! array_filter(config('rolemaster.spell_list_type'), function ($item) use ($validated) {
+            return $item['code'] === $validated['list_type'];
+        })) {
+            return response()->json(['error' => 'unknown list type'], 422);
+        }
+
+        $spellList->description = $validated['description'];
+        $spellList->notes = $validated['notes'];
+        $spellList->list_type = $validated['list_type'];
+        $spellList->name = $validated['name'];
+
+
+        if ($spellList->save()) {
+            return response()
+                ->json(
+                    ['data' => json_decode($spellList->toJson(JSON_PRETTY_PRINT))],
+                    201,
+                    ['Location' => route("spell-lists.show", ['spell' => $spellList->id])]
```

Comment:
He aplicado #59 a tu rama y ahora obtengo un error en esta línea porque dice que le falta un parámetro a `spell-lists.show` llamado `spell_list`.
```suggestion
                    ['Location' => route("spell-lists.show", ['spell_list' => $spellList->id])]
```

Reply:
Cambia el comportamiento por defecto a la hora de generar los alias a rutas?